test/helper.js

```diff
@@ -33,20 +39,30 @@ const ClientExpiration = new Date((new Date()).getTime() + (60 * 1000));
 // Some default clients for the mockAuthServer
 var defaultClients = [
   {
-  clientId:     'captain-write', // can write captain's secrets
-  scopes:       [
-    'secrets:set:captain:*',
-    'secrets:update:captain:*',
-    'secrets:remove:captain:*'
-  ],
-  expiry:      ClientExpiration,
-  credentials:  cfg.get('taskcluster:credentials')
+    clientId:     'captain-write', // can write captain's secrets
+    scopes:       [
+      'secrets:set:captain:*',
+      'secrets:update:captain:*',
+      'secrets:remove:captain:*'
+    ],
+    expiry:       ClientExpiration,
+    credentials:  cfg.get('taskcluster:credentials')
+  }, {
+    clientId:     'captain-read', // can read captain's secrets
+    accessToken:  'none',
+    scopes:       ['secrets:get:captain:*'],
+    expiry:       ClientExpiration,
+    credentials:  cfg.get('taskcluster:credentials')
   }, {
-  clientId:     'captain-read', // can read captain's secrets
-  accessToken:  'none',
-  scopes:       ['secrets:get:captain:*'],
-  expiry:      ClientExpiration,
-  credentials:  cfg.get('taskcluster:credentials')
+    clientId:     'captain-read-write',
+    scopes:       [
+      'secrets:set:captain:*',
+      'secrets:update:captain:*',
+      'secrets:remove:captain:*',
+      'secrets:get:captain:*'
+    ],
+    expiry:       ClientExpiration,
+    credentials:  cfg.get('taskcluster:credentials')
```

Comment:
So this is not really the way to do it...

Note that other things are out-dated in tc-index helper.js, but the way it uses createMockAuthServer is pretty smart:
https://github.com/taskcluster/taskcluster-index/blob/master/test/helper.js#L111-L115

It passes real creds to createMockAuthServer so it can proxy request for access to azure table storage... but everything else it just answers on it's own... So you don't use real credentials in your tests...

I guess we could even extend createMockAuthServer to access azure credentials and then not even proxy, might be nicer for contributors... Granted I think we can anyone access to taskclusterdev, granted they aren't evil :)